coil-base/src/test/java/coil/memory/StrongMemoryCacheTest.kt

```diff
@@ -0,0 +1,121 @@
+package coil.memory
+
+import coil.annotation.ExperimentalCoilApi
+import coil.bitmappool.BitmapPool
+import coil.memory.MemoryCache.Key
+import coil.util.DEFAULT_BITMAP_SIZE
+import coil.util.createBitmap
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import kotlin.test.assertEquals
+import kotlin.test.assertNotNull
+import kotlin.test.assertNull
+
+@RunWith(RobolectricTestRunner::class)
+@OptIn(ExperimentalCoilApi::class)
+class StrongMemoryCacheTest {
```

Comment:
`MemoryCacheTest` -> `StrongMemoryCacheTest`. Looks like I fumbled the rename 🤦.